documentation/openapi.json

```diff
@@ -727,6 +745,67 @@
           }
         }
       }
+    },
+    "/operation/{operationId}": {
```

Comment:
For consistency with the new format, should this be something like "/operation?{operationId}" instead? Looking at the rest of the PR, I'm assuming this hasn't been merged with the main branch in a while since other endpoints are still using the old format, but just something I wanted to make sure I pointed out :)

Reply:
I actually intend to leave these as they are. I don't think these are the same types of requests as our diagnostic based requests, and follow REST semantics more closely. I.e. an operation has a unique id, where as a process is not (and there are several ways to specify a process).